### PR TITLE
Updated Drake & Definitions File to include 2014 HMDA data references

### PR DESCRIPTION
**Pull request by kave**
Wait to merge this pull request when 2014 data becomes publicly available

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,28 @@ To load the HMDA dataset, run `lein repl` and enter the following:
 (in-ns 'qu.loader)
 (load-dataset "hmda")
 ```
-
 Then close your REPL, using `Ctrl-D`.
-
 This does start the web server, so you may want to use a different configuration that starts it on an unpublished port.
+
+## Speeding up data load
+You can take advantage of more CPU and Ram by concurrently loading chunks of data by splitting the large data file
```
Comment (kave) [inline README.md]: @m3brown readme entry for splitting the files

File: resources/datasets/hmda/Drakefile [hunk 2]
```diff
@@ -60,23 +60,26 @@ gaz_counties.zip <- [-timecheck]
...
+  curl http://www.ffiec.gov/hmdarawdata/LAR/National/2007HMDALAR%20-%20National.zip > $OUTPUT7
+
+hmda_lar_all_2014_no_fips.csv <- hmda_lar_2014.zip [method:add_headers_and_unzip]
 hmda_lar_all_2013_no_fips.csv <- hmda_lar_2013.zip [method:add_headers_and_unzip]
 hmda_lar_all_2012_no_fips.csv <- hmda_lar_2012.zip [method:add_headers_and_unzip]
 hmda_lar_all_2011_no_fips.csv <- hmda_lar_2011.zip [method:add_headers_and_unzip]
 hmda_lar_all_2010_no_fips.csv <- hmda_lar_2010.zip [method:add_headers_and_unzip]
 hmda_lar_all_2009_no_fips.csv <- hmda_lar_2009.zip [method:add_headers_and_unzip]
 hmda_lar_all_2008_no_fips.csv <- hmda_lar_2008.zip [method:add_headers_and_unzip]
-hmda_lar_all_2007_no_fips.csv <- hmda_lar_2007.zip [method:add_headers_and_7unzip]
```
Comment (m3brown) [inline resources/datasets/hmda/Drakefile]: `7unzip` is not a typo, there's actually an `add_headers_and_7unzip` function.  Is that function no longer working for you using the 2007 ffiec zipfile?

Comment (kave) [inline resources/datasets/hmda/Drakefile]: @m3brown ah this was an oversight then. i'll revert it

Comment (kave) [inline resources/datasets/hmda/Drakefile]: @m3brown done

**Comment by marcesher** [comment]
@kave This is close but not quite done yet.

Check out https://github.com/cfpb/api/blob/master/resources/datasets/hmda/definition.json#L3 and friends... the date ranges need to be updated to end at 2014

**Pull request merged**